store.html

```diff
@@ -21,7 +21,7 @@
       <img src="./img/karma-logo.svg" alt="karma-logo" />
       <nav>
         <ul>
-          <li>Meet Karma</li>
+          <li style="font-weight: 500">Meet Karma</li>
```

Comment:
Instead of inline CSS, I guess it would be better if we give this list element an ID, and define its style in the css file? This can make you easier to maintain your site.

Ref: https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css

Reply:
updated [uped header li style](https://github.com/CodeYourFuture/HTML-CSS-Module-Project/pull/510/commits/cce672b5ef0eb5dc4fdbd8148446eccf51cb4f83)